cloud.dev.deploy.yaml

```diff
@@ -27,8 +27,20 @@ steps:
       - "--allow-unauthenticated"
 
   - name: "gcr.io/cloud-builders/gcloud"
-    id: "deploy-sweden"
+    id: "migrate-db-sweden"
     waitFor: ["build"]
+    args:
+      - "run"
+      - "jobs"
+      - "execute"
```

Comment:
Going with the execute command now. I couldn't find a way to upsert (create or update) the job nicely, so went with creating the job manually outside CI and then let CI execute it only. It's similar to what we do for Cloud Run servives

Reply:
What happens should the migration job fail?

Reply:
This step return an exit code 1 which prevent the next step (deployment) from being run